deploy/13_test_deployments/01_deploy_referralValidator.ts

```diff
@@ -0,0 +1,19 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
```

Comment:
ReferralValidator doesn't have to be deployed, it's only extended by LandSaleWithReferral or EstateSaleWithAuth

Reply:
fixed -> https://github.com/thesandboxgame/sandbox-smart-contracts/pull/413/commits/c21e258b6910eb27815f6dcf70feba6cd4111bd4